docs/reference/release-notes/8.8.2.asciidoc

```diff
@@ -0,0 +1,43 @@
+[[release-notes-8.8.2]]
+== {es} version 8.8.2
+
+coming[8.8.2]
+
+Also see <<breaking-changes-8.8,Breaking changes in 8.8>>.
+
+[[bug-8.8.2]]
+[float]
+=== Bug fixes
+
+Aggregations::
+* Fix iteration of empty percentiles throwing Null Pointer Exception {es-pull}96668[#96668] (issue: {es-issue}96626[#96626])
+
+Health::
+* Uses `ClusterSettings` instead of Node `Settings` in `HealthMetadataService` {es-pull}96843[#96843] (issue: {es-issue}96219[#96219])
```

Comment:
```suggestion
* Use `ClusterSettings` instead of Node `Settings` in `HealthMetadataService` {es-pull}96843[#96843] (issue: {es-issue}96219[#96219])
```